astropy/stats/sigma_clipping.py

```diff
@@ -194,6 +208,18 @@ def perform_clip(_filtered_data, _kwargs):
         warnings.warn("Input data contains invalid values (NaNs or infs), "
                       "which were automatically masked.", AstropyUserWarning)
 
+    if masked is False and HAS_SCIPY is True:
+        if cenfunc is not np.ma.mean or stdfunc is not np.std:
+            warnings.warn("Using np.mean and np.std to calculate the"
+         "sigma limits, ignoring user defined functions..", AstropyUserWarning)
+
+        filtered_data = sigmaclip(data, sigma_lower, sigma_upper)
```

Comment:
Maybe do a loop here or a message that ``iters`` is ignored.

For ``axis`` there should be also such a message since scipy_sigmaclip ignores that one too.